code/sonalyze/db/sacct_csv_parser.go

```diff
@@ -0,0 +1,355 @@
+// Parser for sacct data.  This is optimized; it uses the same structure as the Sample parser.
+
+package db
+
+import (
+	"bytes"
+	"errors"
+	"io"
+	"math"
+
+	. "sonalyze/common"
+)
+
+func ParseSacctLog(
```

Comment:
"ParseSlurmCSV" I think, to match the file tags that will be introduced, or "ParseSacctCSV" here and then change that file tag.